experimental/packages/otlp-exporter-base/src/platform/browser/OTLPExporterBrowserBase.ts

```diff
@@ -39,10 +40,15 @@ export abstract class OTLPExporterBrowserBase<
     super(config);
     this._useXHR =
       !!config.headers || typeof navigator.sendBeacon !== 'function';
+    if (config.headers && typeof config.headers === 'function') {
```

Comment:
nit I think the first check is not needed here
```suggestion
    if (typeof config.headers === 'function') {
```

Reply:
Thx for the spot, removed now